frontend/src/features/help/constants/HelpText.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactNode } from 'react';
```

Comment:
This component provides structured relationships between Help Pages, Help Topics, Help Text and Help Forms.